code/game/objects/items/flamethrower.dm

```diff
@@ -248,3 +248,25 @@
 
 /obj/item/device/assembly/igniter/proc/ignite_turf(obj/item/flamethrower/F,turf/open/location,release_amount = 0.05)
 	F.default_ignite(location,release_amount)
+
+
+//Upgraded version of the flamethrower for syndie use, identical except for a larger release_amount & air_transfer
+/obj/item/flamethrower/op
+	name = "syndicate flamethrower"
+	desc = "You are a firestarter! Twisted firestarter!"
+	create_full = TRUE
+	create_with_tank = TRUE
+
+
+
+
+/obj/item/flamethrower/op/default_ignite(turf/target, release_amount = 0.075)
```

Comment:
Or you know use variables instead of hard coding an override to the proc. .

Reply:
So you would rather it if I had a var for 'release_amount', then apply that to the proc. Then determine the air_transfer.gases[/datum/gas/plasma][MOLES] from said 'release_amount' var?

This would tie in well with what I had planned actually. I initially intended to have the release quantity adjustable with the syndicate flamethrower having higher max values.